app/views/projects/projects/show.html.erb

```diff
@@ -1,21 +1,85 @@
-<h1><%= @project.title %></h1>
+<div class="row">
+<nav class="projects-nav">
+  <div class="projects-nav__title">
+    <h1><%= @project.title %></h1>
+  </div>
+  <div class="project-nav__menu">
+    <ul class="inline-list">
+      <li>
+        <a href="#">Projects home</a>
```

Comment:
We could remove this and use `projects-nav__title` as the link back home?

Can use the [`link_to_unless_current`](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to_unless_current) helper to prevent this being a link when we're on the projects homepage.

I think we'll want:

```erb
<%= link_to_unless_current @project.title, project_path(@project) %>
```

Or even `<%= link_to_unless_current @project.title, @project %>` might work.

Reply:
`<%= link_to_unless_current @project.title, @project %>` seems to work